src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -125,6 +125,11 @@ MapDocumentActionHandler::MapDocumentActionHandler(QObject *parent)
     mActionToggleOtherLayers->setIcon(
             QIcon(QLatin1String(":/images/16x16/show_hide_others.png")));
 
+    mActionToggleLockOtherLayers = new QAction(this);
+    mActionToggleLockOtherLayers->setShortcut(tr("Ctrl+Shift+U"));
```

Comment:
I guess `Ctrl+Shift+U` was chosen because `Ctrl+Shift+L` is already taken by "Save All"? I wonder if it wouldn't be more intuitive to use `Ctrl+Shift+L` here and maybe drop the "Save All" shortcut again (or switch that one to `Ctrl+Alt+S`, which is what Notepad++ did).

Reply:
Yes I had put the shortcut to Ctrl+Shift+L first. I agree with your suggestion.

Reply:
Sorry, we can have only one `Ctrl+Shift+S` shortcut, and now it means both "Save As" and "Save All" (in which case neither will trigger). Please change "Save As" to `Ctrl+Alt+S`.

Reply:
Hi Bjorn, I can not see that Save As has the "Ctrl+Shift+S" shortcut. Have I overlooked it somewhere?

Reply:
Yes, it happens in this line, because `Ctrl+Shift+S` is a common shortcut for "Save As":
```c++
mUi->actionSaveAs->setShortcuts(QKeySequence::SaveAs);
```